src/diffusers/configuration_utils.py

```diff
@@ -153,11 +154,17 @@ def from_config(cls, pretrained_model_name_or_path: Union[str, os.PathLike], ret
         </Tip>
 
         """
-        config_dict = cls.get_config_dict(pretrained_model_name_or_path=pretrained_model_name_or_path, **kwargs)
-
-        init_dict, unused_kwargs = cls.extract_init_dict(config_dict, **kwargs)
-
-        model = cls(**init_dict)
+        device_map = kwargs.pop("low_cpu_mem_usage", None)
```

Comment:
Ideally we would like to try to keep `configuration_utils.py` framework and component independent. Could we maybe try to set:

```
with accelerate.init_empty_weights():
            model, unused_kwargs = cls.from_config(
            config_path,
            cache_dir=cache_dir,
            return_unused_kwargs=True,
            force_download=force_download,
            resume_download=resume_download,
            proxies=proxies,
            local_files_only=local_files_only,
            use_auth_token=use_auth_token,
            revision=revision,
            subfolder=subfolder,
            device_map=device_map,
            **kwargs,
        )
```

in `modeling_utils.py` instead?

Reply:
Just did it.